hrjobcontract/js/src/job-contract/controllers/contract.js

```diff
@@ -82,10 +82,12 @@ define([
 
         if (isCurrentContract) {
           $scope.$parent.contract.is_current = '1';
+          $scope.$parent.contractCurrent = _.uniq($scope.$parent.contractCurrent);
```

Comment:
if uniq returns more than one contract then it can be `$scope.$parent.currentContracts`